src/main/java/duke/InputHandler.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+public abstract class InputHandler {
+    protected Ui ui;
+    protected TaskList taskList;
```

Comment:
You could add a line between? (To keep your coding style consistent, same thing with couple of other files)